doc/decisions/lowlevel_library.md

```diff
@@ -0,0 +1,61 @@
+# Operations Library
+
+## Problem
+
+There are many functions that can be useful, but are not required to form the minimal API.
+Since we want to keep the API of `libelektra-core` minimal, we cannot add these functions.
+
+If a function is generally useful, it should be added to [`libelektra-operations`](operations_library.md).
+However, some functions are only useful as a C API and shouldn't be part of a language binding.
```

Comment:
Sentence unclear: I am missing the info that `libelektra-operations` should be part of a language binding?